_posts/2021-05-24-design-is-your-friend.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title: "Design is Your Friend: Harnessing the Power of Design (and designers) in Open Source"
```

Comment:
@coderberry  add `permalink: /blog/:title/` below `title: ...`

That eliminates the `/2021/05/24/` uri scheme.